setup/login-alerts.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Create a script to be called when a user logs in
+cat << 'EOF' > /etc/ssh/login-alert.sh
+#!/bin/bash
```

Comment:
how strange.  it appears that the code was changed but my comment remains.

clearly there's nothing wrong with these two comments!